fighthealthinsurance/utils.py

```
@@ -371,7 +374,7 @@ async def execute_critical_optional_fireandforget(
     required_tasks_finished = 0
     # First, execute required tasks (no timeout)
     try:
-        for task in asyncio.as_completed(all_tasks):
+        for task in asyncio.as_completed(all_tasks, timeout=timeout):
```

Ruff (0.8.2)

`timeout` may be undefined, or defined from star imports

(F405)